src/test/java/lotto/domain/model/lotto/LottoNumberTest.java

```diff
@@ -0,0 +1,42 @@
+package lotto.domain.model.lotto;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import lotto.error.ErrorMessage;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class LottoNumberTest {
+
+    @DisplayName("번호가 로또 숫자 범위(1~45)를 벗어나면 예외가 발생한다.")
+    @ParameterizedTest
```

Comment:
`@ParameterizedTest`를 사용하면서도, 각각의 파라미터를 사용한 테스트 이름을 예쁘게 출력할 수 있다는 사실, 알고 계셨나요?!
```suggestion
    @ParameterizedTest(name = "{0}을 사용해 `LottoNumber`를 생성하면 예외가 발생한다.")
```
![image](https://github.com/woowacourse-precourse/java-lotto-6/assets/76612738/352f2ce7-24d3-4fdb-be6e-1ae675e912bb)

Reply:
와 이방법은 진짜 몰랐네요 감사합니다!!